README.md

```diff
@@ -154,6 +154,41 @@ $call = $client->upsertAccount([
 ]);
 ```
 
+#### Add user(s) to an account
+
+```php
+$call = $client->addUsersToAccount([
+    "account" => [
+        "accountId" => "accountId",
+        "domain" => "acme-inc.com",
+        ],
+    "users" => [
+        ["userId" => "1"], // Unique identifier for the user in your database
+        ["userId" => "2"]
+    ],
+]);
+```
+
+#### Remove user(s) from an account
+
+Please note that journy.io makes a difference between removing and deleting:
+
+- Removing: when removing a user, the user will still be stored but marked as "removed".
+- Deleting: when deleting an user, the user will not be stored anymore.
```

Comment:
This is no longer relevant, right? But it's good to add a note that removing a user from an account will still keep a reference to the user.